GameData/ROEngines/PartConfigs/RZ_BDB.cfg

```diff
@@ -0,0 +1,166 @@
+PART
+{
+	module = Part
+	name = ROE-RZ
+	author = CobaltWolf, Pap
+	
+	RSSROConfig = true
+	ROESetEngineDefaults = LIQUID_ENGINE
+	
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+	
+	MODEL
+	{
+		model = ROEngines/Assets/BDB/Thor/bluedog_Thor_LR79
+		scale = 1.502, 1.502, 1.502
+	}
+	
+	scale = 1.0
+	rescaleFactor = 1.0
+	node_stack_top = 0.0, 1.4227, 0.0, 0.0, 1.0, 0.0, 2
+	node_stack_bottom = 0.0, -2.022, 0.0, 0, -1, 0, 2
+	node_attach = 0.0, 1.4227, 0.0, 0.0, 1.0, 0.0, 2
```

Comment:
Being picky, but would you mind resetting the scale to 1.0, using rescaleFactor = 1.502, and dividing out the 1.502 through the three node_* lines to simplify any future rescaling?